src/concrete_minkowski_difference.jl

```diff
@@ -0,0 +1,40 @@
+export minkowski_difference, pontryagin_difference
+
+"""
+    minkowski_difference(Q::LazySet{N}, P::LazySet{N}) where {N<:Real}
+
+Concrete Minkowski difference (geometric difference) for a pair of
+lazy sets.
+
+### Input
+
+- `Q`         -- lazy set
+- `P`         -- another lazy set that is subtracted from `Q`
+
+### Output
+
+An `HPolytope` that corresponds to the Minkowski difference of `Q` minus `P` if
+`Q` is bouneded, an `HPolyhedron` if `Q` is unbouneded.
+
+### Notes
+
+This function requires that the list of constraints of the lazy sets `Q` is
+available and that the set `P` is bounded.
```

Comment:
Can you add an "Algorithm" section that just points to the appropriate theorem from the paper you sent (*) in the issue? for an example of algorithm section see

https://github.com/JuliaReach/LazySets.jl/blob/master/src/Zonotope.jl#L518

(*) https://github.com/JuliaReach/LazySets.jl/issues/1478#issuecomment-513608587